test/end-to-end/test_studio_with_ssl_cert_file_envvar_set.ps1

```diff
@@ -21,7 +21,7 @@ function New-TemporaryDirectory {
 }
 
 Write-Host "--- Generating a signing key"
-hab origin key generate "$env:HAB_ORIGIN"
+hab origin key generate "$HAB_ORIGIN"
```

Comment:
Do we really want to treat this as a local variable and not an environment variable? I don't see where this variable is bwing set.

Reply:
Corrected in 97cb2e787f6ed2015fc99886dcf941b219d0e18d